File: Examples/ClassDiagram/Java/Library.java
package medialibrary;

import java.util.ArrayList;
import java.util.List;

/** The user's whole collection: every playlist in one place. */
public final class Library {
    private final List<Playlist> playlists;

    public Library(List<Playlist> playlists) {
        this.playlists = new ArrayList<>(playlists);
    }

    public List<Playlist> playlists() {
        return playlists;
    }

    public void addPlaylist(Playlist playlist) {
        playlists.add(playlist);
    }

    public Playlist playlist(String named) {
        return playlists.stream().filter(p -> p.name().equals(named)).findFirst().orElse(null);
    }
}

File: Examples/ClassDiagram/Python/library.py
from playlist import Playlist


class Library:
    """The user's whole collection: every playlist in one place."""

    def __init__(self, playlists: list[Playlist] | None = None):
        self.playlists: list[Playlist] = playlists if playlists is not None else []

    def addPlaylist(self, playlist: Playlist) -> None:
        self.playlists.append(playlist)

    def playlist(self, named: str) -> Playlist | None:
        for playlist in self.playlists:
            if playlist.name == named:
                return playlist
        return None
